namespaces/live-1.cloud-platform.service.justice.gov.uk/laa-cla-backend-staging/resources/iam.tf

```diff
@@ -0,0 +1,43 @@
+resource "random_id" "laa-cla-end-to-end-id" {
+  byte_length = 16
+}
+
+resource "aws_iam_user" "laa-cla-end-to-end-user" {
+  name = "laa-cla-end-to-end-user-${random_id.laa-cla-end-to-end-id.hex}"
+}
+
+resource "aws_iam_access_key" "laa-cla-end-to-end-user" {
+  user = aws_iam_user.laa-cla-end-to-end-user.name
+}
+
+data "aws_iam_policy_document" "laa-cla-end-to-end" {
+  statement {
+    actions = [
+      "ecr:BatchCheckLayerAvailability",
+      "ecr:BatchGetImage",
+      "ecr:GetDownloadUrlForLayer",
+    ]
+
+    resources = [
+      "var.cla-public-ecr-arn"
```

Comment:
If I understood correctly, I think you want multiple resources in here? Weren't there 3 or so repos you wanted to pull an image from? Othat than that, looks fine to me :)

Reply:
Yes there are supposed to be three here, but I am just testing with one, when that works I will add the other two